src/image/ExposureFitsReader.cc

```diff
@@ -401,6 +401,10 @@ class ExposureFitsReader::ArchiveReader {
      *
      * @throws pex::exceptions::NotFoundError Thrown if the component is
      *     registered in the file metadata but could not be found.
+     *
+     * @note When accessing via %pybind11, components with derived subclasses, such
```

Comment:
I'd say "from Python"; I'm not sure all our Python developers think of themselves as pybind11 users.

EDIT: sorry for putting my original comment on the commit rather than the PR!